code/setup.dm

```diff
@@ -322,33 +308,107 @@ var/MAX_EXPLOSION_RANGE = 14
 
 //Cant seem to find a mob bitflags area other than the powers one
 
-// bitflags for clothing parts
-#define HEAD			1 		//top of the head
+// bitflags for mob parts
+
+#define HEAD			1
 #define EYES			2048
 #define MOUTH			4096
 #define EARS			8192
-#define FULL_HEAD		14337 //everything
+
 #define UPPER_TORSO		2
 #define LOWER_TORSO		4
 #define LEG_LEFT		8
 #define LEG_RIGHT		16
-#define LEGS			24
 #define FOOT_LEFT		32
 #define FOOT_RIGHT		64
-#define FEET			96
 #define ARM_LEFT		128
 #define ARM_RIGHT		256
-#define ARMS			384
 #define HAND_LEFT		512
 #define HAND_RIGHT		1024
-#define HANDS			1536
-#define FULL_BODY		16383
+
+
+// bitflags for clothing parts
+
+#define CRANIUM			HEAD
+#define FOREHEAD		EYES
+#define FULL_TORSO		UPPER_TORSO|LOWER_TORSO
+#define FACE			FOREHEAD|MOUTH|BEARD
+#define BEARD			32768
+#define FULL_HEAD		HEAD|EYES|MOUTH|EARS|BEARD //everything
+#define LEGS			LEG_LEFT|LEG_RIGHT 		// 24
+#define FEET			FOOT_LEFT|FOOT_RIGHT 	//96
+#define ARMS			ARM_LEFT|ARM_RIGHT		//384
+#define HANDS			HAND_LEFT|HAND_RIGHT //1536
+#define FULL_BODY		FULL_HEAD|HANDS|FULL_TORSO|ARMS|FEET|LEGS //65535
+#define IGNORE_INV		16384 // Don't make stuff invisible
+
+
+// bitflags for preset clothing parts
+// this is used with the variable body_parts_covered, and makes it far easier to set up with less snowflaking.
+
+// MISC CLOTHING
+
+#define CLOTHING_JUMPSUIT	ARMS|LEGS|FULL_TORSO
```

Comment:
Yes and a coder getting it wrong whatever percentage of a time because of the shitty define that obfuscates what it is is UNACCEPTABLE